vignettes/baggr_model_convergence.Rmd

```diff
@@ -0,0 +1,119 @@
+---
+title: "Understanding Model Convergence"
+author: "Brice Green"
+date: "`r Sys.Date()`"
+output:
+  rmarkdown::html_vignette:
+  toc: true
+vignette: >
+  %\VignetteIndexEntry{baggr_model_convergence}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+bibliography: baggr.bib
+---
+
+The `baggr` package makes running Bayesian meta-analysis models very easy, but that ease can also obscure some things that go on behind the scenes. Especially with low-data settings, say with only a few measurements, you may get cryptic error messages when you run your models, like
+
+```
+Warning: There were 110 divergent transitions after warmup. Increasing adapt_delta above 0.9 may help. See
+http://mc-stan.org/misc/warnings.html#divergent-transitions-after-warmup
+```
+
```

Comment:
Here you need to explain basic terms. In few words, as if explaining to an undergraduate. What is Stan - 1 sentence. What is it for. What is MCMC. How does baggr use Stan. Why problems can occur.

Reply:
By "here" I meant after "This vignette" paragraph. But before you go into any particualr issue.

Reply:
I know that you will re-state this in more detail below, but sometimes people stop reading at first difficulty. So having a simple paragraph (it could be a bulletpointed mini-glossary?) will enable many of them.

Reply:
That is useful feedback, I'll make these changes.